### DISCUSSION TITLE
Two Quick Question on Docker Image with two tags?

### DISCUSSION THREAD

**Discussion by quella** [FreshRSS/FreshRSS#4393] [Q&A] [answered]
I am looking at moving my current manual install of FreshRSS from a PI 4 (non-docker) to a Docker Image on a different Pi 4.  

**Question1:** Is there such a thing as an ":edge" version of the ":amd" image?  I would like to continue to be on the development version (edge), but I could not seem to use both tags of :edge and :amd when I tried to pull it.  Am I missing something here, or is the edge version only generated for the Debian images?

**Question2:** Can I tar up my "data" directory from the manual install and place that in my new docker setup, or do I need to start over.  I can export the feeds in OPML (nothing to save in the feeds), but I do have quite a few custom words for marking as read in many of my feeds.  I can start over from scratch if that is needed, but if I can get some of the old install over, I'm happy to do that.

Happy for any suggestions to assist in the transition from manual install to Docker.

**Comment by Frenzie** [comment, marked as answer]
The Debian image is AMD, unless you mean ARM? That's something like edge-arm, see the Docker readme file.

The old install cam be transferred for the most part, on my phone atm though but @Alkarex will probably know the right reference better regardless.

**Reply by quella** [reply]
Sorry, I did mean arm.  I did not try :edge-arm but I did try a few other options that did not work.  Thanks for the pointers and I'm off to play more.  :)

:edge-arm worked!  Thank you very much on getting me to the right image.  I will see what @Alkarex has to say about the ability to pull over the old data folder info.

**Reply by Alkarex** [reply]
For a migration and/or backup, I recommend exporting your user's database with `./cli/export-sqlite-for-user.php` and importing it again in your new instance.

https://github.com/FreshRSS/FreshRSS/blob/edge/cli/README.md

Otherwise, regarding your data folder, you can either mount it as a local folder `-v /path/to-your-freshrss-data:/var/www/FreshRSS/data`, or use a Docker volume such as `-v freshrss_data:/var/www/FreshRSS/data` in which you can copy your existing data with e.g. [`docker cp`](https://docs.docker.com/engine/reference/commandline/cp/)

If you are not reinstalling from scratch, double-check manually your `./data/config.php` as some parameters might need to be changed, such as base_url, database settings...

https://github.com/FreshRSS/FreshRSS/blob/edge/Docker/README.md

**Reply by quella** [reply]
@Alkarex thank you for providing the additional details.  I found it easier to have both the manual install and docker container up in a browser window and manually copied the settings from one to the other.  I was not as concerned with the actual feed data only making sure the transition kept me from missing anything in my various feeds.  Thank you again for the effort and great tool I use multiple times a day.
